joomla/joomla-cms
# [searchtools] Simplify also com_templates styles and templates views

### Summary of Changes

This PR does the same for com_templates styles and templates views as done in https://github.com/joomla/joomla-cms/pull/10090 for:
- com_languages installed view
- com_menus items view
- com_modules modules view
- com_cache cache view
- com_checkin checkin view

In other words all the other list views that have a selector.

### Testing Instructions

1. Apply patch
2. Confirm that com_templates styles and templates views searchtools  work correctly

### Documentation Changes Required

None.
